api/src/main/resources/workbench.yaml

```diff
@@ -470,3 +494,50 @@ definitions:
       phoneNumber:
         description: the user's phone number
         type: string
+
+  CriteriaListResponse:
+    type: "object"
+    required:
+      - "items"
+    properties:
+      items:
+        type: "array"
+        items:
+          $ref: "#/definitions/Criteria"
+
+  Criteria:
+    type: "object"
+    required:
+      - "id"
+      - "type"
+      - "code"
+      - "name"
+      - "group"
+      - "selectable"
+    properties:
+      id:
+        description: id of the criteria
+        type: integer
+        format: int64
+      type:
+        description: type of criteria
+        type: string
+      code:
+        description: what level of data access the user has
+        type: string
+      name:
+        description: description of criteria
+        type: string
+      count:
+        description: est. count in the cdr
+        type: integer
+        format: int64
+      group:
+        description: specifies of child or parent
```

Comment:
"child" or "parent"? 

Is the idea that nodes which are "parent" are not selectable, and are just containers for children?

Reply:
some are not selectable and some are depending on the specific criteria tree, so we use a combination of isGroup and IsSelectable to determine what is a parent or leaf and is selectable

Reply:
So "group" = "true" means there are children, and "selectable" = "true" means you can query on it?

Reply:
yes